### PR TITLE
chore(description-list): removed unnecessary column fill examples

### PR DESCRIPTION
**Pull request by mcoker**
fixes https://github.com/patternfly/patternfly/issues/4592

### CODE REVIEW

**Review by mattnolting — COMMENTED** [review]
Initially, it was difficult for me to understand what was happening in this example. It would be beneficial to add a description for here. Maybe something like:

> Column fill will modify the default placement of list groups from left-to-right, to top-to-bottom using css `column-count`. Note: using this modifier will break horizontal alignment provided by css grid.

Otherwise, LGTM

**Comment by mcoker** [comment]
@mattnolting updated

File: src/patternfly/components/DescriptionList/examples/DescriptionList.md [hunk 1]
```diff
@@ -57,18 +57,11 @@ cssPrefix: pf-c-description-list
...
 ### Column fill
-```hbs
-{{> description-list__example description-list--title="Column fill" description-list--modifier="pf-m-fill-columns"}}
-```
 
-### Column fill, horizontal
-```hbs
-{{> description-list__example description-list--title="Column fill, horizontal" description-list--modifier="pf-m-fill-columns pf-m-horizontal"}}
-```
+Column fill will modify the default placement of description list groups to fill from top to bottom using css column-count, instead of left to right. **Note:** using this modifier will change the layout so that horizontally adjacent groups are no longer aligned in the same row.
```
Comment (mattnolting) [inline src/patternfly/components/DescriptionList/examples/DescriptionList.md:61]: Beautifully put ❤️

**Review by mattnolting — APPROVED** [review]
Nice work, looks great 💪

**Pull request merged**